src/components/Convertation.tsx

```diff
@@ -0,0 +1,14 @@
+import { Movie } from '../types/Movie';
+import { MovieData } from '../types/MovieData';
+
+export const convert = (data: MovieData): Movie => {
```

Comment:
It would be great to make function's name more descriptive, e.g. 'createMovieFromData' or 'convertDataIntoMovie'